frontend/src/utility/WebSocketMessageUtils.js

```diff
@@ -0,0 +1,14 @@
+export function createChatMessage(message) {
```

Comment:
I was thinking this might be a cleaner way to have utility functions, as opposed to mixins. I think the way I was using mixins before wasn't the intended use case.

Reply:
I don't have an opinion on this. I think the other utility function you made, the one that accepts api and payload is sufficient though